src/assets/i18n/en.json

```diff
@@ -567,5 +567,12 @@
   "MORE_CONTENT": "Find more books and content on {{%s}} below.",
   "SEARCH_BOOK": "Search for content (Eg: ’English Stories’)",
   "TEXTBOOK": "Textbooks",
-  "EXTRACTING_CONTENT": "Extracting content"
+  "EXTRACTING_CONTENT": "Extracting content",
+  "MERGE_ACCOUNT": "Merge Account",
+  "ACCOUNT_MERGE_CONFIRMATION_HEADING": "Merge Account",
+  "ACCOUNT_MERGE_CONFIRMATION_CONTENT": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec dictum est eu orci sollicitudin faucibus.",
```

Comment:
Add a valid  value for ACCOUNT_MERGE_CONFIRMATION_CONTENT like "Do you like to merge account?"

Reply:
Resolved: Used valid message - "Would you like to merge account?"